Kickstarter-iOS/Views/Cells/PledgeAmountCell.swift

```diff
@@ -134,6 +136,25 @@ final class PledgeAmountCell: UITableViewCell, ValueCell {
   }
 }
 
+extension PledgeAmountCell: UITextFieldDelegate {
+  func textField(
+    _ textField: UITextField, shouldChangeCharactersIn _: NSRange, replacementString string: String
+  ) -> Bool {
+    let decimalSeparatorCharacters = CharacterSet.ksr_decimalSeparators()
+    let existingCharacters = CharacterSet(charactersIn: textField.text ?? String.init())
+    let inputCharacters = CharacterSet(charactersIn: string)
+    let numericCharacters = CharacterSet.ksr_numericCharacters()
+
+    if numericCharacters.isSuperset(of: inputCharacters) {
```

Comment:
Potentially one less return if I have the logic correct:
```swift
guard decimalSeparatorCharacters.isSuperset(of: inputCharacters) else {
  return numericCharacters.isSuperset(of: inputCharacters)
}

return !decimalSeparatorCharacters.isSubset(of: existingCharacters)
```

Reply:
Unfortunately your condition doesn't support deleting characters so I think I'll keep it the way it is for now.